deployment/packer/template.js

```diff
@@ -4,7 +4,9 @@
         "branch": "",
         "aws_region": "",
         "aws_ubuntu_ami": "",
-        "stack_type": ""
+        "stack_type": "",
+        "postgresql_password": "",
```

Comment:
I don't think changes here will make Packer fail if variables aren't present. If we want to do that, I think it would have go to into the CLI tool.